addons/binding/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/config/PowermaxIpConfiguration.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.powermax.internal.config;
+
+/**
+ * The {@link PowermaxIpConfiguration} is responsible for holding
+ * configuration informations associated to a Powermax IP thing type
+ *
+ * @author Laurent Garnier
+ */
+public class PowermaxIpConfiguration {
+
+    public String ip;
+    public Integer tcpPort;
```

Comment:
Yes getter's would make sense as well, but meant whether it would make sense to use primitive `int/boolean` for `motionOffDelay, allowArming, allowDisarming, forceStandardMode, autoSyncTime`

Reply:
Ok, sorry, I misunderstood what you asked.
In fact, I used Integer rather than int for example because I thought this is what is expected to match a thing configuration. For any thing configuration, parameters could be provided or not and that is probably the reason to use Objects rather than primitives.